core/src/banking_stage.rs

```diff
@@ -702,36 +615,26 @@ impl BankingStage {
                     .increment_consume_buffered_packets_us(consume_buffered_packets_us);
             }
             BufferedPacketsDecision::Forward => {
-                let ((), forward_us) = measure_us!(forwarder.handle_forwarding(
-                    unprocessed_transaction_storage,
-                    false,
-                    slot_metrics_tracker,
-                    banking_stage_stats,
-                ));
-                slot_metrics_tracker.increment_forward_us(forward_us);
-                // Take metrics action after forwarding packets to include forwarded
-                // metrics into current slot
-                slot_metrics_tracker.apply_action(metrics_action);
+                // get current working bank from bank_forks, use it to sanitize transaction and
```

Comment:
@AshwinSekar is this a reasonable spot to do this instead of where we did previously?

Reply:
yep this is fine, we just need to run at least once an epoch